client/app/states/services/details/details.html

```diff
@@ -56,8 +56,9 @@
                   <li role="menuitem"><a href="#" ng-click="vm.retireServiceLater()" ng-show="vm.showScheduleRetirementService" translate>Schedule Retirement</a></li>
                 </ul>
               </div>
-              <button class="btn btn-primary" type="button" ng-click="vm.editServiceModal()" ng-show="vm.showEditService" translate>Edit Service</button>
-              <button class="btn btn-primary" type="button" ng-click="vm.ownershipServiceModal()" ng-show="vm.showSetOwnership" translate>Set Ownership</button>
+              <button class="btn btn-primary" type="button" ng-click="vm.editServiceModal()" translate>Edit Service</button>
+              <button class="btn btn-primary" type="button" ng-click="vm.ownershipServiceModal()" translate>Set Ownership</button>
+              <button class="btn btn-primary" type="button" ng-if="vm.service.reconfigure" ng-click="vm.reconfigureService(vm.service.id)" translate>Reconfigure</button>
             </div>
```

Comment:
@h-kataria Looks like the RBAC check was removed from here as well ...specifically the  ```ng-show``` for ```Edit Service``` and ```Set Ownership``` seems to be deleted and which brings me to the topic of having RBAC support for ```Reconfigure``` -- we would need that as well, right?